engine/src/witness/btc_e.rs

```diff
@@ -103,7 +106,11 @@ impl VoterApi<BitcoinVaultDepositWitnessingES> for BitcoinVaultDepositWitnessing
 		_settings: <BitcoinVaultDepositWitnessingES as ElectoralSystemTypes>::ElectoralSettings,
 		properties: <BitcoinVaultDepositWitnessingES as ElectoralSystemTypes>::ElectionProperties,
 	) -> Result<Option<VoteOf<BitcoinVaultDepositWitnessingES>>, anyhow::Error> {
-		let (witness_range, vaults, _extra) = properties;
+		let BWElectionProperties {
+			block_height: witness_range,
+			properties: vaults,
+			reorg_id: _,
```

Comment:
nit, we can just use `..`
```suggestion
			properties: vaults,
            ..,
```

Reply:
same below too ofc